deploy/olm-catalog/manifests/lighthouse.submariner.io_serviceimports.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: serviceimports.lighthouse.submariner.io
```

Comment:
should we use upstream group?

Reply:
not sure upstream group can work or not, I will try it

Reply:
submariner-operator 0.7.0 cannot support upstream api, we will update this later